utils/tf.py

```diff
@@ -31,7 +34,7 @@ class TFFrozenModelRunner:
     """
     A class providing facilities to run TensorFlow frozen model (in frozen .pb format).
     """
-    def __init__(self, path_to_model: str, output_names: list):
+    def __init__(self, path_to_model: str, output_names: list, use_opsgraph=False):
```

Comment:
Hmmm could this ops graph trickery be taken out of this generic class perhaps? Maybe you could swap self.session from run.py of wide_deep after initializing the class with the opsgraph one? I haven't seen any other model needing this functionality.